packages/components/psammead-navigation/src/index.jsx

```diff
@@ -157,11 +179,68 @@ const StyledNav = styled.nav`
   }
 `;
 
-const Navigation = ({ children, dir }) => (
-  <StyledNav role="navigation" dir={dir}>
-    <NavWrapper>{children}</NavWrapper>
-  </StyledNav>
-);
+const SwipeableNav = styled.div`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    position: relative;
+    overflow-x: scroll;
+    white-space: nowrap;
+    scroll-behavior: smooth;
+    -webkit-overflow-scrolling: touch;
```

Comment:
Tested this on iOS 13 and the navigation doesn't scroll.. If anyone has an idea why, please shout it out.